src/flashbar/interfaces.ts

```diff
@@ -40,6 +40,69 @@ export namespace FlashbarProps {
     warningIconAriaLabel?: string;
   }
 
+  export interface Style {
+    item?: {
+      root?: {
+        background?: {
+          error?: string;
+          info?: string;
+          inProgress?: string;
+          success?: string;
+          warning?: string;
+        };
+        borderColor?: {
+          error?: string;
+          info?: string;
+          inProgress?: string;
+          success?: string;
+          warning?: string;
+        };
+        borderRadius?: string;
+        borderWidth?: string;
+        color?: {
```

Comment:
What's the plan for dealing with visual contexts? Currently, e.g. a link inside a flashbar gets the incorrect color 
![Screenshot 2025-06-12 at 11 53 44](https://github.com/user-attachments/assets/98ccead8-ca8f-495c-a918-654aee72511e), applies for various other tokens too: https://github.com/cloudscape-design/components/blob/scottob-flashbar-style-api/style-dictionary/visual-refresh/contexts/flashbar.ts#L9